roles/setup_gitops/tasks/local-registry.yml

```diff
@@ -33,4 +33,20 @@
           - mirrors:
               - "{{ sg_local_registry }}/rhacm2/multicluster-operators-subscription-rhel9"
             source: "{{ sg_mce_repo }}"
+
+- name: Apply IDMS for ZTP and MCE images
+  kubernetes.core.k8s:
+    definition:
+      apiVersion: config.openshift.io/v1
+      kind: ImageDigestMirrorSet
+      metadata:
+        name: gitops-images
+      spec:
+        imageDigestMirrors:
+          - mirrors:
+              - "{{ sg_local_registry }}/openshift4/ztp-site-generate-rhel8"
+            source: "{{ sg_ztp_repo }}"
+          - mirrors:
+              - "{{ sg_local_registry }}/rhacm2/multicluster-operators-subscription-rhel9"
+            source: "{{ sg_mce_repo }}"
```

Comment:
LGTM, but wdyt of adding this as a dci_workaround for [OCPBUGS-59221](https://issues.redhat.com/browse/OCPBUGS-59221)